packages/bot/src/scratch/hooks/block_svg.js

```diff
@@ -82,30 +82,25 @@ Blockly.BlockSvg.prototype.showContextMenu_ = function(e) {
     // Option to collapse/expand block.
     if (this.workspace.options.collapse) {
         if (this.collapsed_) {
-            const expandOption = { enabled: true };
-            expandOption.text = localize('Expand Block');
-            expandOption.callback = function() {
-                block.setCollapsed(false);
-            };
-            menuOptions.push(expandOption);
+            const expand_option = { enabled: true };
+            expand_option.text = localize('Expand Block');
+            expand_option.callback = () => block.setCollapsed(false);
```

Comment:
@aaron-binary I think for arrow functions you can do something like this:
```suggestion
            expand_option.callback = (() => this.setCollapsed(false)).bind(this);
```